src/components/Participant/UpdateParticipant/UpdateParticipant.tsx

```diff
@@ -200,21 +200,35 @@ const UpdateParticipant = () => {
     handlePageTitle(currentPage);
   }, [currentPage]);
 
+  const capitalizeWords = (input: string) => {
+    return input
```

Comment:
returns can be implemented in one of 3 ways.  

- The `return` keyword 
- Wrapping in parens `()` instead of curlys `{}`  this is more common for JSX
- Or if only one item simply following after the fat arrow 

So in your case consider point 3